docs/developer/setup.rst

```diff
@@ -79,6 +84,25 @@ Once minikube is started, make sure you can access it by running:
     NAME       STATUS    ROLES     AGE       VERSION
     minikube   Ready     <none>    3s        v1.10.0
 
+**Notice**: If minikube didn't start successfully issue ``minikube delete`` before next ``minikube start``.
+
+Once minikube is started issue:
+
+.. code-block:: console
+
+    $ minikube addons enable coredns
+    $ eval $(minikube docker-env)
+
+Installing Chartpress
+^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: console
+
+    $ pip install chartpress
+    $ chartpress --tag latest
```

Comment:
This isn't about installing chartpress, but about building the images. So this is probably unnecessary here. But you may say two words at the beginning of this section about chartpress, e.g.

"We use the `chartpress` package to bundle together images and charts."

Reply:
Yes, that's true but I guess `pip install chartpress` should stay here maybe?

Reply:
Actually, we use `pipenv` anyway with the provided `Pipfile` -- that's what the `make minikube-deploy` will use. So this section is not really necessary IMO.

Reply:
I'm sure you know better how the Python machinery works here. I took the section out.